.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+# Summary of Changes
+- Include a summary of changes made by this pull request
+
+## Pull Request Checklist
+
+### Is Content created by the story?
+- [ ] No
+- [ ] If yes, the content is:
+   - [ ] Accessible
+   - [ ] Mobile-friendly
+   - [ ] Version-controlled
+   - [ ] Governed by appropriate roles and permissions
+   - [ ] Organized by taxonomy
+
+### Are there User-facing changes?
+- [ ] No
+- [ ] If yes, then:
+   - [ ] User documentation is updated
+   - [ ] User training is updated
+
+### Are there Upstream Code changes?
+- [ ] No
+- [ ] If yes, then:
+   - [ ] Code is complete, commented, and mergeable against *develop* branch.
+   - [ ] Code is secure to the best of our knowledge and does not contain anything that cannot be on a public repo (e.g. passwords)
+   - Is a test necessary?
+      - [ ] No
+      - [ ] If yes, then:
+         - [ ] Add a test if existing tests do not effectively test the code change
+         - [ ] Test builds without error
```

Comment:
Does this mean "test passes"?

Reply:
Yes - updated to read "Test passes" instead.